source/core/serial_solid.cc

```diff
@@ -48,6 +45,7 @@ SerialSolid<dim, spacedim>::SerialSolid(
   , this_mpi_process(Utilities::MPI::this_mpi_process(mpi_communicator))
   , param(param)
   , id(id)
+  , output_bool(&param->output_bool)
```

Comment:
no need for a pointer with an adress, just pass the value itself. I don't see the point of having this pool pointer floating around